tests/unit/test_bugjira.py

```diff
@@ -0,0 +1,180 @@
+from unittest.mock import Mock, create_autospec
+from xmlrpc.client import Fault
+
+import pytest
+from bugzilla import Bugzilla
+from jira import JIRA
+from jira.exceptions import JIRAError
+
+import bugjira.broker as broker
+from bugjira.bugjira import Bugjira
+from bugjira.issue import BugzillaIssue, JiraIssue
+
+
+@pytest.fixture(scope="function", autouse=True)
+def setup(monkeypatch):
+    """Patch out the bugzilla.Bugzilla and jira.JIRA constructors
+    in the broker module so that we don't attempt to connect to an actual
+    backend
```

Comment:
I don't remember if the last rev had this comment, but nice addition either way!